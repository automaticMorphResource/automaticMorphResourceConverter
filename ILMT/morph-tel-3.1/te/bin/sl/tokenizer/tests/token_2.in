కానీ ఆస్ట్రేలియా ఖండాన్ని చూసిన తరువాత... ఎవరో చిత్రకారుడు ప్రశాంతమైన మనసుతో కుదురుగా కూర్చుని గీసిన బొమ్మంత ఎర్రగా, అందంగా ఉంది కదూ.. అనిపిస్తుంది. 
