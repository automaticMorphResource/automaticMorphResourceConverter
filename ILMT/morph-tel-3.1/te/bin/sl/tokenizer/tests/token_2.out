<Sentence id="1">
1	కానీ	unk
2	ఆస్ట్రేలియా	unk
3	ఖండాన్ని	unk
4	చూసిన	unk
5	తరువాత...	unk
6	ఎవరో	unk
7	చిత్రకారుడు	unk
8	ప్రశాంతమైన	unk
9	మనసుతో	unk
10	కుదురుగా	unk
11	కూర్చుని	unk
12	గీసిన	unk
13	బొమ్మంత	unk
14	ఎర్రగా	unk
15	,	unk
16	అందంగా	unk
17	ఉంది	unk
18	కదూ..	unk
19	అనిపిస్తుంది	unk
20	.	unk
</Sentence>

